jax/_src/numpy/lax_numpy.py

```diff
@@ -4733,9 +4733,33 @@ def array_equiv(a1: ArrayLike, a2: ArrayLike) -> Array:
 
 # General np.from* style functions mostly delegate to numpy.
 
-@util.implements(np.frombuffer)
 def frombuffer(buffer: bytes | Any, dtype: DTypeLike = float,
                count: int = -1, offset: int = 0) -> Array:
+  """Convert a buffer into a 1-D JAX array.
+
+  JAX implementation of :func:`numpy.frombuffer`.
+
+  Args:
+    buffer: an object containing the data.
```

Comment:
We should say what kind of input is supported; I believe it must be either a `bytes` object with length that is an integer multiple of the `dtype` element size, or it must be an object exporting the [Python buffer interface](https://docs.python.org/3/c-api/buffer.html).

Reply:
description added.